lib/proxy/helpers.rb

```diff
@@ -25,7 +25,8 @@ def log_halt code=nil, exception=nil
 
   # read the HTTPS client certificate from the environment and extract its CN
   def https_cert_cn
-    certificate_raw = request.env['SSL_CLIENT_CERT'].to_s
+    ssl_client_cert_header = Proxy::SETTINGS.ssl_client_cert_header || 'SSL_CLIENT_CERT'
+    certificate_raw = request.env[ssl_client_cert_header].to_s.gsub(/   /, "\n")
```

Comment:
Hmm, according to https://www.w3.org/Protocols/rfc2616/rfc2616-sec2.html (see section 2.2 Basic Rules, look for "folded onto multiple lines" line) header continuation uses either a single space or a tab. Are you trying to handle something that's nginx-specific?